### PR TITLE
TTL changes

### PR DESCRIPTION
**Pull request by hannanelson**

### CODE REVIEW

File: modules/ROOT/pages/osv2-apis.adoc [hunk 1]
```diff
@@ -4,21 +4,22 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: osv2, os2, object store, store, rest, apis
 
-Anypoint Object Store v2 (Object Store v2) provides a REST API that lets you get and store `key,list` values.
+Anypoint Object Store v2 (Object Store v2) provides a REST API that enables you to get and store `key,list` values.
```
Comment (alegmarra) [inline modules/ROOT/pages/osv2-apis.adoc]:
```suggestion
Anypoint Object Store v2 (Object Store v2) provides a REST API that enables you to get and store `key,value` pairs.
```

File: modules/ROOT/pages/osv2-apis.adoc [hunk 2]
```diff
@@ -33,7 +34,7 @@ The following table compares the Object Store REST API operations to Object Stor
 |POST |
 
 * Object Store:
-** TTL in seconds for the object store - by default the TTL is 2,592,000 seconds (30 days). See xref:osv2-faq.adoc#how-long-can-data-persist-in-object-store-v2[How long can data persist in Object Store v2?]
+** TTL in seconds for the object store. The maximum TTL is 2,592,000 seconds (30 days). See xref:osv2-faq.adoc#how-long-can-data-persist-in-object-store-v2[How long can data persist in Object Store v2?]
```
Comment (alegmarra) [inline modules/ROOT/pages/osv2-apis.adoc]: 30 days is both the maximum AND the default.

**Review by alegmarra — CHANGES_REQUESTED** [review]

**Review by jjayakumaran — APPROVED** [review]

**Review by alegmarra — APPROVED** [review]

**Review by prafulrana — APPROVED** [review]

File: modules/ROOT/pages/osv2-apis.adoc [hunk 3]
```diff
@@ -33,7 +34,7 @@ The following table compares the Object Store REST API operations to Object Stor
 |POST |
 
 * Object Store:
-** TTL in seconds for the object store - by default the TTL is 2,592,000 seconds (30 days). See xref:osv2-faq.adoc#how-long-can-data-persist-in-object-store-v2[How long can data persist in Object Store v2?]
+** TTL in seconds for the object store. The default and maximum TTL is 2,592,000 seconds (30 days). See xref:osv2-faq.adoc#how-long-can-data-persist-in-object-store-v2[How long can data persist in Object Store v2?]
```
Comment (kbostrom) [inline modules/ROOT/pages/osv2-apis.adoc]:
```suggestion
** TTL, in seconds, for the object store. The default and maximum TTL is 2,592,000 seconds (30 days). See xref:osv2-faq.adoc#how-long-can-data-persist-in-object-store-v2[How long can data persist in Object Store v2?]
```
OR
```suggestion
** TTL for the object store, in seconds. The default and maximum TTL is 2,592,000 seconds (30 days). See xref:osv2-faq.adoc#how-long-can-data-persist-in-object-store-v2[How long can data persist in Object Store v2?]
```

File: modules/ROOT/pages/osv2-faq.adoc [hunk 4]
```diff
@@ -49,7 +49,17 @@ The total size of an object store is not limited, but each value is limited to 1
 
 == How long can data persist in Object Store v2?
 
-Object Store v2 data persists 30 days. The expiry time is reset if an update to that key occurs. If there is no update, the key is deleted at the 30 day mark.
+The maximum TTL (time to live) is 2,592,000 seconds (30 days).
```
Comment (kbostrom) [inline modules/ROOT/pages/osv2-faq.adoc:52]: Why is TTL spelled here and not in others? Does this appear "first" (although that can be hard to determine)?

File: modules/ROOT/pages/osv2-faq.adoc [hunk 5]
```diff
@@ -49,7 +49,17 @@ The total size of an object store is not limited, but each value is limited to 1
...
+
+* If you set the TTL in the app, the data persists for the time specified. 
+
+* If you don't set the TTL in the app, the TTL depends on data access:
+
+** If the data is accessed at least once a week in the 30 days after key creation, the TTL is extended for 30 days.
```
Comment (kbostrom) [inline modules/ROOT/pages/osv2-faq.adoc]: extended from...?

File: modules/ROOT/pages/osv2-guide.adoc [hunk 6]
```diff
@@ -133,7 +133,8 @@ image::osv2-ui.png[]
 you can delete a partition or the store itself by hovering and clicking the trash
 can icon.
 
-*Note:* Object Store provides persistent storage of objects with a time to live (TTL) of 30 days.
+[NOTE]
+Object Store v2 provides persistent storage of objects with a time to live (TTL) of up to 30 days.
```
Comment (kbostrom) [inline modules/ROOT/pages/osv2-guide.adoc]: Does "with a time to live..." modify storage or objects?

File: modules/ROOT/pages/osv2-tutorial.adoc [hunk 7]
```diff
@@ -3,7 +3,7 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-This document helps you use Object Store v2 to store and view data.
+Use this tutorial to get started with using Object Store v2 to store and view data.
```
Comment (kbostrom) [inline modules/ROOT/pages/osv2-tutorial.adoc]: We could avoid two "use"s by something like "This tutorial helps you to start using ...."

File: modules/ROOT/pages/osv2-tutorial.adoc [hunk 8]
```diff
@@ -437,7 +437,8 @@ image::osv2-ui.png[]
 you can delete a partition or the store itself by hovering and clicking the trash
 can icon.
 
-*Note:* Object Store v2 provides persistent storage of objects with a time to live (TTL) of up to 30 days.
+[NOTE]
+Object Store v2 provides persistent storage of objects with a time to live (TTL) of up to 30 days.
```
Comment (kbostrom) [inline modules/ROOT/pages/osv2-tutorial.adoc]: ditto prev. question about this sentence.

**Review by kbostrom — APPROVED** [review]
Here you go!

File: modules/ROOT/pages/osv2-faq.adoc (hunk 4 above)
Comment (hannanelson) [reply modules/ROOT/pages/osv2-faq.adoc:52]: first instance in this topic.

File: modules/ROOT/pages/osv2-tutorial.adoc (hunk 8 above)
Comment (hannanelson) [reply modules/ROOT/pages/osv2-tutorial.adoc]: Changed to: Objects are stored for up to 30 days, based on the TTL (time to live).

File: modules/ROOT/pages/osv2-guide.adoc (hunk 6 above)
Comment (hannanelson) [reply modules/ROOT/pages/osv2-guide.adoc]: Changed to: Objects are stored for up to 30 days, based on the TTL (time to live).

File: modules/ROOT/pages/osv2-faq.adoc (hunk 5 above)
Comment (hannanelson) [reply modules/ROOT/pages/osv2-faq.adoc]: Changed to "extended for another 30 days" and added clarification.

Comment (jjayakumaran) [reply modules/ROOT/pages/osv2-faq.adoc]: extended for another 30 days from the current date of expiry

**Review by jjayakumaran — APPROVED** [review]

**Pull request merged**